ckan/logic/action/create.py

```diff
@@ -1574,3 +1575,42 @@ def follow_group(context, data_dict):
         follower=follower.follower_id, object=follower.object_id))
 
     return model_dictize.user_following_group_dictize(follower, context)
+
+
+def api_token_create(context, data_dict):
+    """Create new API Token for current user.
+
+    You must provide your API key in the Authorization header.
+
+    :returns: a representation of the 'api_token'
```

Comment:
You need to document the input parameters in the data dict, `user` (user name or id?) and `name`.

Can you describe the returned key here? eg: Returns a dict with the key `token` containing the encoded token value.
Is it worth describing the internal decoded structure (ie `jti` and `iat` keys), I guess not as this is an internal implementation detail

Reply:
Updated. 
Yes, `jti` and `iat` are implementation-specific and generally, they won't be used by end-user directly. You are just copying encoded token and using it as is. Probably, the majority of users won't even know, that there are some data inside the token